### ISSUE TITLE
Alerts should not be focusable if it can't be interacted with

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#1087]
On the Alerts page: https://modus-web-components.trimble.com/?path=/docs/components-alert--default#alert the alerts are focusable (with keyboard tabbing) but the ones without buttons (non-interactive) should not be. Non-interactive elements should not be focusable.

Screenshot showing Default alert with focus state:
![image](https://user-images.githubusercontent.com/1212885/223459763-3f87b3d1-4cbd-448d-869f-15c63d4efec9.png)

If the alert has a close/dismiss button maybe just that should be focusable as is done with Bootstrap.
e.g. https://getbootstrap.com/docs/5.3/components/alerts/#dismissing
![image](https://user-images.githubusercontent.com/1212885/223460341-88c2b3c1-08b1-42d6-a914-6e40f38a6230.png)

**Comment by msankaran0712** [comment]
@coliff We have it on production now, https://modus-web-components.trimble.com/?path=/story/components-alert--dismissible
